Taskfile.yml

```diff
@@ -101,16 +104,30 @@ tasks:
         "{{.G_CORE_COMPONENT_BUILD_DIR}}/clp"
         "{{.G_CORE_COMPONENT_BUILD_DIR}}/clp-s"
         "{{.G_CORE_COMPONENT_BUILD_DIR}}/reducer-server"
-        "{{.G_WEBUI_NODEJS_BIN_DIR}}/node"
         "{{.OUTPUT_DIR}}/bin/"
+      - >-
+        rsync -a
+        "{{.G_WEBUI_NODEJS_BIN_DIR}}/node"
+        "{{.OUTPUT_DIR}}/bin/node-14"
+      - >-
+        rsync -a
+        "{{.G_LOG_VIEWER_WEBUI_BIN_DIR}}/node"
+        "{{.OUTPUT_DIR}}/bin/node-22"
       - "mkdir -p '{{.OUTPUT_DIR}}/var/www/'"
       - >-
         rsync -a
         "{{.G_WEBUI_BUILD_DIR}}/"
-        "{{.OUTPUT_DIR}}/var/www/"
+        "{{.OUTPUT_DIR}}/var/www/webui/"
       - |-
-        cd "{{.OUTPUT_DIR}}/var/www/programs/server"
+        cd "{{.OUTPUT_DIR}}/var/www/webui/programs/server"
         PATH="{{.G_WEBUI_NODEJS_BIN_DIR}}":$PATH npm install
+      - >-
+        rsync -a
+        "{{.G_LOG_VIEWER_WEBUI_BUILD_DIR}}/"
+        "{{.OUTPUT_DIR}}/var/www/log_viewer/"
+      - |-
+        cd "{{.OUTPUT_DIR}}/var/www/log_viewer/server"
+        PATH="{{.G_LOG_VIEWER_WEBUI_BIN_DIR}}":$PATH npm i
```

Comment:
We should probably add the [`--production` flag](https://docs.npmjs.com/cli/v10/commands/npm-install#:~:text=With%20the%20%2D%2Dproduction%20flag) to prevent installations of the [devDependencies](https://github.com/y-scope/clp/blob/6e2166592869e70aa603acbfeab53df396f70e29/components/log-viewer-webui/server/package.json#L23).
With that said, I didn't find the related description on the https://docs.npmjs.com/cli/v10/commands/npm-ci page. If the flag is `npm install` specific, we shall explore the `--omit` or `--include` options.

Reply:
Looking at the specification, if `NODE_ENV` is set to production, both npm install and npm clean-install will not install dev dependencies [(here)](https://docs.npmjs.com/cli/v10/commands/npm-ci#omit).
We can set NODE_ENV in the task to achieve this, but it also make sense to use the omit flag explicitly. let me know what do you think.

Reply:
Good point. I'm fine with both the NODE_ENV and "--production" approches. For consistency with our other Node.js commands, the NODE_ENV approach seems better.